pino/pino.d.ts

```diff
@@ -1,86 +1,90 @@
-// Type definitions for pino v1.0.1
+// Type definitions for pino v2.12.3
 // Project: https://github.com/mcollina/pino.git
 // Definitions by: Peter Snider <https://github.com/psnider>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
-/// <reference path="../node/node.d.ts"/>   // only for require()
+/// <reference path="../node/index.d.ts"/>   // only for require()
 
 declare module 'pino' {
+    import stream = require('stream');
 
-    import stream = require('stream')
+    namespace pino {
```

Comment:
@Volox I just submitted an update to pino.d.ts to bring in line with the latest version.
See: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/13303